library/azure_rm_roleassignment_info.py

```diff
@@ -135,12 +138,16 @@ def __init__(self):
             ),
             assignee=dict(
                 type='str'
+            ),
+            role_definition_id=dict(
+                type='str'
```

Comment:
i saw below, there's:
 "elif self.resource_group:"
but i can't see resource_group in parameters anywhere, should we have it?